hooks/pre-commit

```diff
@@ -3,14 +3,23 @@
 jsfiles=$(git diff --cached --name-only --diff-filter=ACM "*.js" "*.jsx" "*.ts" "*.tsx" | tr '\n' ' ')
 [ -z "$jsfiles" ] && exit 0
 
-# Check if any of the files are ugly
+# Check if any of the files are ugly or contain a console.log call
 uglyFiles=()
+consoleFiles=""
 for jsfile in $jsfiles; do
     diff "$jsfile" <($(npm bin)/prettier "$jsfile") >/dev/null || uglyFiles+=("$jsfile")
+    if [ "$(git diff --cached "$jsfile" | grep '^+.*console.log' -c)" -gt '0' ] ; then
+        consoleFiles="$consoleFiles '$jsfile'"
+    fi
 done
 
-if [ -n $uglyFiles ]; then
+if [ -n "$uglyFiles" ]; then
     echo "Prettify your files first:"
     echo '$(npm bin)/prettier --write' "${uglyFiles[@]}"
```

Comment:
We should have a script e.g. `npm run pretty` that will handle this, and mention it here.

(I realise this bit of code is not your fault but I don't want to forget about this)

Reply:
Would this really be useful though? Since the necessary command is displayed contributors can just copy/paste it to run it.

Reply:
> Would this really be useful though?

Yeah, I want something that I can run before committing so that the commits are never rejected by the hook.